roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1917,8 +1917,8 @@ def __init__(self, role, filename, local_facts,
             self.system_facts = ansible_facts(module, ['hardware', 'network', 'virtual', 'facter'])  # noqa: F405
             for (k, v) in self.system_facts.items():
                 self.system_facts["ansible_%s" % k.replace('-', '_')] = v
-        except UnboundLocalError:
-            # ansible-2.2
+        except (UnboundLocalError, RuntimeError):
```

Comment:
Yeah, this definitely feels heavy handed to me. The error these both are addressing is:

```
Traceback (most recent call last):
  File "/tmp/ansible_487jx9mk/ansible_module_openshift_facts.py", line 2491, in <module>
    main()
  File "/tmp/ansible_487jx9mk/ansible_module_openshift_facts.py", line 2478, in main
    protected_facts_to_overwrite)
  File "/tmp/ansible_487jx9mk/ansible_module_openshift_facts.py", line 1899, in __init__
    for (k, v) in self.system_facts.items():
RuntimeError: dictionary changed size during iteration
```
which I believe would be `UnboundLocalError` in python 2 and the above `RuntimeError` in python 3. Since we don't support `ansible-2.1` (?) we may just want to remove the try.